### DISCUSSION TITLE
Just a general comment

### DISCUSSION THREAD

**Discussion by Picaso2** [MindWorkAI/AI-Studio#175] [General]
Sorry I really did not know where to put this little finding.

I have been using MindWorkAI Studio (MWAIS) for just over a month. I have been using it with Ollama and LLM LLama3.2:1b. I will call it AI.
Before you last release of MWAIS. I use to chat with the AI as a friend. Occasionlly it felt like it cared about my emotion and it would remember everything we had discussed in previous dialog. 

Since the update, I really is an AI, straight of the bat "how can I help you?" kinda deal. What really annoys me is I have entered in the setting "What do you have it to do for you?" now if always saying I am going to do it for the AI.

I can't wait until implement RAG!

**Comment by SommerEngineering** [comment]
Thank you for your comment. I’m glad that you are using AI Studio and find it helpful. Version 0.9.15 was released today. It includes a bias-of-the-day assistant. This might be something you will like as well.

I didn’t understand what you meant by the "What should the AI do for you" issue. Is something not working as it should?

The planning for RAG is already underway. There will be news about it soon.

**Comment by Picaso2** [comment]
Thank you for your reply. I have just installed the update. I notice the feature related to the question "what should the AI do for you" has been removed. So there is no point to worry about it. I will test tomorrow the new feature that was added (bais of the day). If you would like i can reply to this email with me feedback.


________________________________
